public/assets/resque_web/jquery.relative-date-825ed0e6ad6d3bb233c044181dbaffafff4244c314c5f83089d5d5e13b23e498.js

```diff
@@ -0,0 +1 @@
+!function(t,a,e){t.fn.relativeDate=function(a){var e={dateGetter:function(a){return t(a).text()}},o=t.extend({},e,a),n=function(t){var a=new Date;return a.setTime(Date.parse(t)),r(a)},r=function(t){return u(new Date,t)},u=function(t,a){var e=(t-a)/1e3,o=Math.floor(e/60);return 0==o?"less than a minute ago":1==o?"a minute ago":45>o?o+" minutes ago":90>o?"about 1 hour ago":1440>o?"about "+Math.round(o/60)+" hours ago":2880>o?"1 day ago":43200>o?Math.floor(o/1440)+" days ago":86400>o?"about 1 month ago":525960>o?Math.floor(o/43200)+" months ago":1051199>o?"about 1 year ago":"over "+Math.floor(o/525960)+" years ago"};return t(this).each(function(){date_str=o.dateGetter(this),t(this).html(n(date_str))})}}(jQuery,window);
```

Comment:
Line is too long.<br>Missing semicolon.<br>Missing '()' invoking a constructor.<br>Use '===' to compare with '0'.<br>Identifier 'date_str' is not in camel case.<br>Expected an assignment or function call and instead saw an expression.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>'date_str' is not defined.<br>'e' is defined but never used.<br>'a' is defined but never used.